MobiusCore/Test/EffectHandlers/EffectRouterDSLTests.swift

```diff
@@ -100,43 +162,26 @@ class EffectRouterDSLTests: QuickSpec {
                 dslHandler.accept(.effect2)
                 expect(events).to(equal([.eventForEffect1, .eventForEffect2]))
             }
-
-            it("Supports routing to a connectable") {
-                var events: [Event] = []
-                let dslHandler = EffectRouter<Effect, Event>()
-                    .routeEffects(equalTo: .effect1).to(EffectConnectable(emitsEvent: .eventForEffect1))
-                    .routeEffects(equalTo: .effect2).to(EffectConnectable(emitsEvent: .eventForEffect2))
-                    .asConnectable
-                    .connect { events.append($0) }
-
-                dslHandler.accept(.effect1)
-                expect(events).to(equal([.eventForEffect1]))
-                dslHandler.accept(.effect2)
-                expect(events).to(equal([.eventForEffect1, .eventForEffect2]))
-            }
         }
 
         context("Effect routers based on payload extracting functions") {
             it("Supports routing to an effect handler") {
                 var events: [Event] = []
                 var wasDisposed = false
-                let effectHandler = EffectHandler<Effect, Event>(
-                    handle: { effect, dispatch in
-                        expect(effect).to(equal(.effect1))
-                        dispatch(.eventForEffect1)
-                    },
-                    disposable: AnonymousDisposable {
-                        wasDisposed = true
-                    }
-                )
+
                 let payload: (Effect) -> Effect? = { $0 == .effect1 ? .effect1 : nil }
                 let dslHandler = EffectRouter<Effect, Event>()
-                    .routeEffects(withPayload: payload).to(effectHandler)
+                    .routeEffects(withPayload: payload).to { effect, response in
+                        expect(effect).to(equal(.effect1))
+                        response.send(.eventForEffect1)
```

Comment:
Should this call `end`? Probably doesn't matter for the purposes of this PR. 

I am starting to think that maybe there should in fact be a way to send a single event, because it is easier. It's breaking the single responsibility principle, but following the principle "don't make the client do anything the library could" (https://dl.acm.org/doi/10.1145/1176617.1176622). But that is something we can address later, no need to do it in this PR.

Reply:
This is testing that the `EffectRouter` is disposed. It will not be disposed it it was ended (which is tested separately).It is confusing though, because this is also testing that it can send events.

I will split this into two test cases to address this confusion

Reply:
`toEvent` is what you would use for sending a single event :)

Reply:
Re `toEvent`: yes, if you know up front that it'll always be a single event. Maybe there are conditions where it'll be one, none, or many. Again, I think we should leave that for later. The existence of `toEvent` makes it even less urgent to address, and when we have some usage of this version, we'll know more about whether we'll want a special-cased solution for single event/end in the callback as well.

Reply:
Ok, I understand. Sounds reasonable. Adding a version of `end` which takes an event would be a compatible change - so, like you said, let's revisit this